### ISSUE TITLE
[Bug] Default language is always incorrect (English) unless you deactivate English language modules

### ISSUE THREAD

**Issue by Wikinaut** [Piwigo/Piwigo#815]
The interface language is - different from what has been chosen during the installation (German) - always English (unless you fully deactivate English languages).

This is also the case when the language for the guest user is set to German.

**Comment by flop25** [comment]
hello
be more specific, the process to select the language is complex:
the language displayed is the one of the user if it's a normal/admin account

the guest and the generic accounts get the lang from the browser

So try to tune your local config
// Registering process and guest/generic members get language from the browser
// if language isn't available PHPWG_DEFAULT_LANGUAGE is used as previously
$conf['browser_language'] = true;

then come back here

**Comment by Wikinaut** [comment]
@flop25 no, it is not. My language settings are all set to German, so there is not a single reason that Piwigo shows English for non-logged-in users.

the guest and the generic accounts get the lang from the browser .... this is GERMAN, but not set.

**Comment by Wikinaut** [comment]
It is set to ```$conf['browser_language'] = true;```

**Comment by Wikinaut** [comment]
gallery/includes/config_default.inc.php

**Comment by flop25** [comment]
please use the Edit Function and/or wait before sending multiple message
You're currently decreasing the readability of your messages and our energy to help you

Avoid word in full caps also. Thank you

**Comment by flop25** [comment]
if you got to http://www.planete-flop.fr/gallerie/ what's the language you get? (the identification block in the menu at the end)
